flocker/node/agents/test/test_blockdevice.py

```diff
@@ -698,6 +752,186 @@ def add_application_with_volume(node_state):
                                   mountpoint=FilePath(b"/data")))})
 
 
+_BLOCKDEVICE_PREFIX = "blockdevice-"
```

Comment:
This is necessary because tests manually construct state. Arguably that's a mistake in test design since it's brittle. Probably not worth spending any more time on this so let's stick to this approach, but at least write comment explaining motivation and suggestion for improvement (always make state using API as in realistic usage?)